src/readii/process/images/crop.py

```
@@ -0,0 +1,428 @@
+    -------
+    cropped_image : sitk.Image
+        Cropped image.
+    """
+    # Check that the number of dimensions in the crop dimensions matches the number of dimensions in the image
+    validate_new_dimensions(image, resize_dimensions)
+
+    # Check that the number of bounding box dimensions match the image dimensions
+    validate_new_dimensions(image, int(len(bounding_box)/2))
+
+    # Current bounding box dimensions
+    current_image_dimensions = (max_x - min_x, max_y - min_y, max_z - min_z)
```

Ruff (0.8.2)

Undefined name `max_x`

(F821)